desktop/libs/indexer/src/indexer/api3_tests.py

```diff
@@ -104,3 +104,22 @@ def test_col_names():
   assert_true('date_1_' in columns_name)
   assert_true('hour_1' in columns_name)
   assert_true('minute' in columns_name)
+
+
+def test_guess_format_excel_remote_file():
+  if sys.version_info[0] > 2:
```

Comment:
Much better to use `raise Skiptest` for the Py2 condition.

Reply:
done